### ISSUE TITLE
AttributeError: 'ExtractEdges' object has no attribute 'points'

### ISSUE THREAD

**Issue by giswangsj** [enthought/mayavi#689]
Hello,
when I run the example [delaunay_graph.py ](https://github.com/enthought/mayavi/blob/master/examples/mayavi/advanced_visualization/delaunay_graph.py)
I get the following error:
`new_x, new_y, new_z = edges.outputs[0].points.to_array().T
AttributeError: 'ExtractEdges' object has no attribute 'points'`
in line 96
I use python3.6 , vtk8.1 and mayavi-4.6.1.
please help me.

**Comment by prabhuramachandran** [comment]
Thanks for the report.  This is a bug!  I'll fix it shortly, in the meanwhile you can replace lines 96 and 97 with the following:
```
    dataset = edges.get_output_dataset()
    new_x, new_y, new_z = dataset.points.to_array().T
    lines = dataset.lines.to_array()
```
The fix is in #691. If you wait for a bit I'll merge this and if you are using master the fix should be available.

**Comment by giswangsj** [comment]
It works, thank you
